ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -711,6 +713,10 @@ function check_upgrade_status() {
             echo -e "Upgrade succeed on $(date "+%F %T")\n\n"
             return 0
         fi
+        if [[ $(oc get nodes -l node.openshift.io/os_id=rhel) != "" ]] && [[ "${UPGRADE_RHEL_WORKER_BEFOREHAND}" == "true" ]] && [[ ${avail} == "True" && ${progress} == "True" && ${out} == *"Unable to apply ${cluster_version}" ]]; then
```

Comment:
I think `[[ "${UPGRADE_RHEL_WORKER_BEFOREHAND}" == "true" ]]` already indicated this is a rhel worker upgrade, so I guess we can drop `[[ $(oc get nodes -l node.openshift.io/os_id=rhel) != "" ]]`

Reply:
Updated